app/controllers/setup_controller.rb

```diff
@@ -65,6 +65,9 @@ def set_roles
   def bootstrap
     @apiserver = Pillar.value(pillar: :apiserver) ||
       Minion.find_by(role: Minion.roles[:master]).fqdn
+    # TODO: Minion.find_by(role: Minion.roles[:admin]).fqdn ?
+    @dashboard_external_fqdn = Pillar.value(pillar: :dashboard_external_fqdn) ||
+      ""
```

Comment:
The admin as a regular minion is included in https://github.com/kubic-project/velum/pull/299/files#diff-fa4c43e571bbfe7bd1ae914e5794a39fR13 (despite ignored in the UI, but important for checking its update status and other general status -- would allow us to do `Minion.admin.first.fqdn`). With our current code and in this specific version I'd set as placeholder `Pillar.value(pillar: :dashboard)` instead of `""`. In the general case it will be a sensible default, but you can still change it depending on what environment you are going to deploy for (e.g. openstack).